packages/govuk-frontend-review/src/app.mjs

```diff
@@ -9,6 +11,9 @@ import * as middleware from './common/middleware/index.mjs'
 import * as nunjucks from './common/nunjucks/index.mjs'
 import * as routes from './routes/index.mjs'
 
+const require = createRequire(import.meta.url)
+const { version } = require('govuk-frontend/package.json')
```

Comment:
@romaricpascal As discussed, we're using the local `govuk-frontend/package.json` not the shared config. This means the review app can still run `govuk-frontend@4` if installed into the workspace

I've had to use `require()` since ES modules get an experimental warning when running:

```mjs
import pkg from 'govuk-frontend/package.json' assert { type: 'json' }
```

Import assertions were reconsidered as [import attributes](https://github.com/tc39/proposal-import-attributes#readme) due to [security concerns](https://github.com/WICG/webcomponents/issues/839)